packs/php/php/config.yml

```diff
@@ -4,8 +4,33 @@ name: PHP
 
 # Description of the pack
 description: |+
-  This is an observability pack for PHP applications.
-  This pack relies on the New Relic PHP agent, with distributed tracing enabled (https://docs.newrelic.com/docs/agents/php-agent/features/distributed-tracing-php-agent/)
+  # Complete Solution to PHP Monitoring
```

Comment:
another pretty wordy description from a developer POV, asking @bradleycamacho to review.

Reply:
I made some copyedits for brevity